clients/banking/src/graphql/partner.gql

```diff
@@ -737,6 +738,7 @@ query CardPage($cardId: ID!) {
       totalCount
     }
     physicalCard {
+      __typename
```

Comment:
Since this PR we don't need anymore to add __typename in graphql queries: https://github.com/swan-io/swan-partner-frontend/pull/105  
The codegen automatically add it everywhere